### DISCUSSION TITLE
「开源之史」系列之一/八：协作的演化——从原子个体到全球开放协作 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#81] [Announcements]
# 「开源之史」系列之一/八：协作的演化——从原子个体到全球开放协作 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/06-the-evolution-of-open-collaboration/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

这次的抽象有点儿乱了,
根据 "人类简史" 的整理, 认为协作基础是共同故事, 而不是政治, 经济其实不过是一种共同故事, 有关价值的..

协作中, 红楼的例子不如 圣经 --> 这可是经过几百年, 上万人共同编撰出来的,
如果这方面有可能和谐, 也建议使用 鸠摩罗什 的大规模翻译工程来比喻...

参考: [佛经翻译 - 维基百科，自由的百科全书](https://zh.wikipedia.org/zh-hans/%E4%BD%9B%E7%B6%93%E7%BF%BB%E8%AD%AF)

其实, 人类的协作从开始到现在, 目标从来没变过: `提高效能`/`凝聚知识`

只不过, 以往只能通过物质媒介来进行协作, 也就是说, 虽然看起来是在共同制造物理上的部件,
其实真正在协作的, 是其中包含的经验和信息;

也就是电脑制造出来之后, 我们的信息终于可以一定程度上脱离物理世界存在了,
软件行业中的协作, 才显的有些另类,
可其实, 从无改变, 只是经验有了更加直接和纯粹的载体;

那么, 根据 呼吸 (Exhalation: Stories)
https://book.douban.com/subject/34672176/
中的阐述, 意识如不断被微风吹动的散布在空间中无数小金属片的颤动间而构成的动态结构;

则未来协作, 必然向意识空间中的直接感应进发, 
嘦我们能找到对潜意识和主意识的可定制保护, 
那么, 工作意识就可以安全在约定意识大厅中高速协作...

这个场景早在 94年黄易小说 星际浪子 中就有过描述

**Reply by lijiangsheng1** [reply]
您的评论让我想到了海洋法法律条款的协作过程。这也是一个伟大的协作过程，值得去挖掘。
